### PR TITLE
Separate apps

### PR DESCRIPTION
**Pull request by wilmhit**

### CODE REVIEW

File: easywarehouse/employee/views.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from django.shortcuts import render
+
+# Create your views here.
```
Comment (songofroland) [inline easywarehouse/employee/views.py:3]: I think we don't need an app for employees now, let's add it when required

File: easywarehouse/images/apps.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class ImagesConfig(AppConfig):
+    default_auto_field = "django.db.models.BigAutoField"
```
Comment (songofroland) [inline easywarehouse/images/apps.py:5]: I think it's not needed when we declare the id explicilty

**Review by songofroland — APPROVED** [review]
Well done

I've did not check the logic tho

File: easywarehouse/employee/views.py (hunk 1 above)
Comment (wilmhit) [reply easywarehouse/employee/views.py:3]: It's required for what I do on other branch

**Pull request merged**